packages/salesforcedx-vscode-core/src/commands/forceSourceDeploy.ts

```diff
@@ -31,39 +39,76 @@ export abstract class ForceSourceDeployExecutor extends SfdxCommandletExecutor<
       : '';
     const execFilePathOrPaths = response.data;
     const execution = new CliCommandExecutor(this.build(response.data), {
-      cwd: workspacePath
+      cwd: workspacePath,
+      env: { SFDX_JSON_TO_STDOUT: 'true' }
```

Comment:
Forcing all output to stdout for the execution

Reply:
Very cool